doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -134,6 +134,10 @@ Programmable proof search
       Behaves like :tacn:`eauto` but shows the tactics it tries to solve the goal,
       including failing paths.
 
+   .. tacn:: dfs eauto {? @nat_or_var } {? @auto_using } {? @hintbases }
+
+      An alias for :tacn:`eauto`.
```

Comment:
BTW, now that we've removed `bfs eauto`, we should deprecate `dfs eauto`. It is probably unused, and it doesn't particularly make sense to keep it around.

Reply:
See #15379